sdk/trace/src/main/scala/org/typelevel/otel4s/sdk/trace/SdkTraceScope.scala

```diff
@@ -0,0 +1,335 @@
+    * │  └─────────────────┘  │        │  └─────────────────┘    └─────────────────┘  │
+    * │                       │        │                                              │
+    * └───────────────────────┘        └──────────────────────────────────────────────┘
+    * }}}
+    *
+    * ===The [[SpanContext]] is valid===
+    *
+    * We override the existing span context with the given one. The baggage data
+    * remains.
+    *
+    * {{{
+    * ┌──────────────────────────────────────────────┐       ┌──────────────────────────────────────────────┐
+    * │   Context                                    │       │   Context                                    │
+    * │                                              │       │                                              │
+    * │  ┌─────────────────┐    ┌─────────────────┐  │       │  ┌─────────────────┐    ┌─────────────────┐  │
+    * │  │                 │    │                 │  │       │  │                 │    │                 │  │
+    * │  │   SpanContext   │    │     Baggage     │  │  ---> │  │   SpanContext   │    │     Baggage     │  │
+    * │  │ span_id = "old" │    │                 │  │       │  │ span_id = "new" │    │                 │  │
+    * │  └─────────────────┘    └─────────────────┘  │       │  └─────────────────┘    └─────────────────┘  │
+    * │                                              │       │                                              │
+    * └──────────────────────────────────────────────┘       └──────────────────────────────────────────────┘
+    * }}}
+    *
+    * ===The [[SpanContext]] is invalid===
+    *
+    * The propagation logic is no-op. We keep the context as is. The baggage
+    * data remains.
+    *
+    * {{{
+    * ┌──────────────────────────────────────────────┐       ┌──────────────────────────────────────────────┐
+    * │   Context                                    │       │   Context                                    │
+    * │                                              │       │                                              │
+    * │  ┌─────────────────┐    ┌─────────────────┐  │       │  ┌─────────────────┐    ┌─────────────────┐  │
+    * │  │                 │    │                 │  │       │  │                 │    │                 │  │
+    * │  │   SpanContext   │    │     Baggage     │  │  ---> │  │   SpanContext   │    │     Baggage     │  │
+    * │  │     invalid     │    │                 │  │       │  │     invalid     │    │                 │  │
+    * │  └─────────────────┘    └─────────────────┘  │       │  └─────────────────┘    └─────────────────┘  │
+    * │                                              │       │                                              │
+    * └──────────────────────────────────────────────┘       └──────────────────────────────────────────────┘
+    * }}}
+    *
+    * @param spanContext
+    *   the span context to use
+    */
+  def makeScope(spanContext: SpanContext): F[F ~> F]
```

Comment:
in `Tracer`, the equivalent method is called `childScope`—should this be called `childScope` as well?

Reply:
Good point. The `childScope` fits better here